CharlieBackend.Data/Repositories/Impl/AccountRepository.cs

```diff
@@ -16,29 +16,29 @@ public AccountRepository(ApplicationContext applicationContext)
         { 
         }
 
-        public Task<Account> GetAccountCredentials(AuthenticationDto authenticationModel)
+        public async Task<Account> GetAccountCredentials(AuthenticationDto authenticationModel)
         {
-            return _applicationContext.Accounts
+            return await _applicationContext.Accounts
                 .FirstOrDefaultAsync(account 
                          => account.Email == authenticationModel.Email 
                                 && account.Password == authenticationModel.Password);
         }
 
-		public Task<Account> GetAccountCredentialsById(long id)
+		public async Task<Account> GetAccountCredentialsById(long id)
```

Comment:
Could you replace these two tabs with spaces and align this method with the others?

Reply:
Could you please specify which row should I align and relatively to what? I might be blind but it's looking pretty much aligned already.

Reply:
The row number 27.  It starts with 2 tabs. They should be replaced with 8 spaces.

Reply:
Done, thanks.